numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -2990,7 +2990,7 @@ array_set_ops_function(PyObject *NPY_UNUSED(self), PyObject *NPY_UNUSED(args),
 {
     PyObject *oldops = NULL;
 
-    if ((oldops = PyArray_GetNumericOps()) == NULL) {
+    if ((oldops = _PyArray_GetNumericOps()) == NULL) {
```

Comment:
Doesn't this mean that you're actually _not_ deprecating the python-side APIs, only the C ones?

Reply:
Since this function is mapped to python-level `set_numeric_ops`, I only want the `DeprecationWarning` from `PyArray_SetNumericOps`, three lines down. There is no python-level `get_numeric_ops`.